cmp/grid/GridModel.js

```diff
@@ -408,6 +409,19 @@ export class GridModel {
         this.applyColumnStateChanges(colStateChanges);
     }
 
```

Comment:
+ I  might put these comments in a block within the method, rather than a jsdoc on the method.
+ Also would support applying to noteAgColumnStateChanged() for symmetry

Reply:
Cool - I had flipped it to a doc comment but don't feel strongly.  I mainly wanted to expand on it a bit as it's the kind of thing I come back to and extra clarity always helps.

These model methods that are called from the component are in kind of an in-between state vis-a-vis public/private, but in general more like a private method for the entire system.  I.e. no reason for app developers to call (arguing for the block comment).